.github/workflows/ci.yml

```diff
@@ -0,0 +1,78 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - master
```

Comment:
Do we have a master branch in this repo? I couldn't find one. If not, could we get rid of this reference?